### DISCUSSION TITLE
Platform mode and Fee

### DISCUSSION THREAD

**Discussion by FonsBiemans** [alfio-event/alf.io#1371] [Q&A]
Hi, 

I'm currently testing with Alf.io to charge different fees for commercial organizations versus community events. I've configured the platform fee at the system, organization, and event levels, and enabled it accordingly. However, when I create a test event and sell tickets (using bank transfer), I cannot find any report of the fees being applied. Here are the details:

Platform Fee Configuration:
- Set Fees at the system level  
- set different fees between organisations.
- Configured event-specific fees.

Problem:
After creating a test event and selling some tickets, the fee is not reported anywhere in the system.
I've checked the event billing reports, but there is no fee column. or any other report i can find. (is there one?)

Requirements:
- I need to ensure that commercial organizations are billed a fixed fee for each sold ticket.
- For free events, no fee should be charged.
- One organization has both commercial and community events, so the configuration needs to be able to handle this differentiation.
- Could you please guide me on where I might find the fee details and reports or if there's any additional configuration I need to perform? Any insights or steps to troubleshoot this issue would be greatly appreciated.

Thank you!

Fons Biemans

**Comment by cbellone** [comment]
Hi @FonsBiemans ,

thank you for using alf.io!

> However, when I create a test event and sell tickets (using bank transfer), I cannot find any report of the fees being applied.

At the moment, fees are applied only when using Stripe, see https://stripe.com/connect and Mollie, see https://www.mollie.com/products/connect. 
As long as you use those payment providers, you should be able to charge organizations. 
For other payment methods (paypal, wire transfer, on site) it's not possible to apply fees, as we wouldn't have any way to charge the organization.
What we _could_ do is implement a Reservation [Extension hook](https://alf.io/docs/reference/extensions/reference/reservation/) and let you call external services to charge organizers.

Happy to review a PR for this, or to discuss about a sponsored development.


> After creating a test event and selling some tickets, the fee is not reported anywhere in the system.

At the moment there is no such report about applied fees.
However, application fees are already persisted in the database: table `b_transaction`, column `plat_fee` (fee will be saved as `cents` amount).

Happy to review a PR for this, or to discuss about a sponsored development.


Celestino
